flyteadmin/pkg/errors/errors.go

```diff
@@ -140,3 +140,8 @@ func NewWorkflowExistsIdenticalStructureError(ctx context.Context, request *admi
 	}
 	return statusErr
 }
+
+func IsDoesNotExistError(err error) bool {
```

Comment:
Does it make sense to name this `IsNotFoundError` since we're checking for the `NotFound` code?

Reply:
I chose the same name as the one in datacatalog for consistency.

Reply:
I referred to https://github.com/flyteorg/flyte/blob/211b80e40ea7c6fda0345beadee45f57486c4848/datacatalog/pkg/errors/errors.go#L62

Reply:
sounds reasonable.